### PR TITLE
Parallelize e2e browserstack tests

### PR DESCRIPTION
**Pull request by mattsoulanille**
Use GNU parallel to run browserstack tests in parallel.

To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

- - -
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/7531)

### CODE REVIEW

**Review by chunnienc — APPROVED** [review]

**Comment by chunnienc** [comment]
I will merge #7532 and resolve conflicts there after this PR is merged.

**Comment by mattsoulanille** [comment]
CI will likely fail since this PR updates the dockerfile, so [here's a link to running CI on the new docker image](https://console.cloud.google.com/cloud-build/builds/5d29e7ee-eb7e-41bc-b9ad-4665f1661538?project=834911136599).

**Comment by mattsoulanille** [comment]
> I will merge #7532 and resolve conflicts there after this PR is merged.

Sounds good to me. Thanks!

**Comment by mattsoulanille** [comment]
> CI will likely fail since this PR updates the dockerfile, so ~[here's a link to running CI on the new docker image](https://console.cloud.google.com/cloud-build/builds/5d29e7ee-eb7e-41bc-b9ad-4665f1661538?project=834911136599)~.
> 
> Once this passes, I'll push this docker image as the new release docker image and re-run CI tests.
> 
> Edit: [New link](https://console.cloud.google.com/cloud-build/builds/fbd2917b-2ef2-4707-9fc8-73a1147b5ecf?project=834911136599) because the previous one included verdaccio tests, which would run the failing golden tests that are disabled in #7532.

I've published the new docker and will re-trigger CI.

File: e2e/scripts/test-ci.sh [hunk 1]
```diff
@@ -44,25 +44,27 @@ cd ..
...
+    "yarn run-browserstack --browsers=bs_android_10 --tags '$TAGS'"
+    # Test script tag bundles
+    "karma start ./script_tag_tests/tfjs-core-cpu/karma.conf.js --browserstack --browsers=bs_chrome_mac"
+  )
+fi
 
-  node ../scripts/run_flaky.js "yarn run-browserstack --browsers=bs_firefox_mac --tags '$TAGS'"
-  node ../scripts/run_flaky.js "yarn run-browserstack --browsers=bs_chrome_mac --tags '$TAGS'"
-  node ../scripts/run_flaky.js "yarn run-browserstack --browsers=bs_android_10 --tags '$TAGS'"
+for t in "${COMMANDS[@]}"; do
```
Comment (chunnienc) [inline e2e/scripts/test-ci.sh:66]: nit: `t` -> `command`

Comment (mattsoulanille) [reply e2e/scripts/test-ci.sh:66]: Thanks!

**Review by pyu10055 — APPROVED** [review]
> Reviewed 2 of 3 files at r1, 1 of 1 files at r2, 4 of 4 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/7531)* status: :shipit: complete! 2 of 1 approvals obtained (waiting on @chunnienc)

**Pull request merged**